packages/client/src/utils/apolloClient.ts

```diff
@@ -39,7 +38,7 @@ export const createClient = (
   restoredCache?: InMemoryCache
 ) => {
   const httpLink = createHttpLink({
-    uri: resolve(window.config.API_GATEWAY_URL, 'graphql')
+    uri: new URL('graphql', window.config.API_GATEWAY_URL).toString()
```

Comment:
The parameters should be other way around. With new URL the first one is the path, and the second one is the base URL.